src/labels/CustomLabels.labels

```diff
@@ -2694,6 +2702,13 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Button text for copying existing Batch setup to a new Batch.</shortDescription>
         <value>COPY</value>
     </labels>
+    <labels>
+        <fullName>bgeDryRunComplete</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Success message when mass dry run completes in BGE.</shortDescription>
+        <value>Mass Dry Run Complete</value>
```

Comment:
Let's work with docs team and product to update the language here with their suggestions